### ISSUE TITLE
[oracle] Parsing issue with XMLATTRIBUTES

### ISSUE THREAD

**Issue by osarier** [sqlfluff/sqlfluff#8112]
### Search before asking

- [x] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### What Happened

Linting the example below leads to a 'Found unparsable section' error.

### Expected Behaviour

The example should be linted correctly.
See https://docs.oracle.com/en/database/oracle/oracle-database/26/sqlrf/XMLELEMENT.html

### Observed Behaviour

`Line 5, Position 18: Found unparsable section: 'as "attr_name1"/n        ,/'attr2/' as "att...'`

### How to reproduce

```sql
select
   xmlelement(
      "xml_el"
      , xmlattributes (
         'attr1' as "attr_name1"
        ,'attr2' as "attr_name2"
      )
   )
from ex_tab;

``` 

### Dialect

oracle

### Version

4.2.1
python 3.11.3

### Configuration

[sqlfluff]
dialect = oracle

### Are you willing to work on and submit a PR to address the issue?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)
